packages/stk/stk_emend/cmake/Dependencies.cmake

```diff
@@ -1,9 +1,9 @@
 SET(LIB_REQUIRED_DEP_PACKAGES STKUtil)
 SET(LIB_OPTIONAL_DEP_PACKAGES)
-SET(TEST_REQUIRED_DEP_PACKAGES Gtest)
+SET(TEST_REQUIRED_DEP_PACKAGES)
 SET(TEST_OPTIONAL_DEP_PACKAGES)
 SET(LIB_REQUIRED_DEP_TPLS MPI)
 SET(LIB_OPTIONAL_DEP_TPLS)
 SET(TEST_REQUIRED_DEP_TPLS)
-SET(TEST_OPTIONAL_DEP_TPLS)
+SET(TEST_OPTIONAL_DEP_TPLS gtest)
```

Comment:
Should that be `REQUIRED`?

Reply:
Yes, since it was a required package it should also be a required TPL.